test/e2e/specs/editor/various/post-visibility.spec.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post visibility', () => {
+	const wp = '';
```

Comment:
Why is this needed?

Reply:
We are using this variable below to get the status of the posts. So it's just declaration of that variable.

Reply:
Hi @kevin940726,

We are using this variable below to get the status of the posts. So it's just declaration of that variable.

Reply:
I believe we can just call `window.wp` instead. Will that work?